csrc/scheduler/matmul_heuristic.h

```diff
@@ -370,6 +370,11 @@ class MatmulParams : public HeuristicParams {
     }
   } cluster_dims;
 
+  //! This is used for persistent kernels only i.e. when tiling_strategy is not
+  //! OneTilePerCTA. This allows us to control the number of CTAs launched in
+  //! order to guarantee only a single wave is launched.
+  int64_t num_clusters = -1L;
```

Comment:
A single cluster seems like a safer default.
```suggestion
  int64_t num_clusters = 1L;
```

Reply:
Yeah, I think they both have their advantages
- The advantage of `1` is that the default value is a launchable kernel 
- The advantage of `-1` is that it signals an incomplete specification so we can error out

I went with -1 to avoid silently getting terrible perf if we forget to set it properly.